bin/docker_push

```diff
@@ -1,43 +1,25 @@
 #!/bin/bash -e
 
-# Travis CI seems to source this script, so it is ignoring the #! above!
```

Comment:
Why remove?

Reply:
I'm not seeing this problem so I removed it to avoid confusions.

Reply:
But now -e is specified in two places with not reasoning... so maybe take out the "set -e" line as well.

Reply:
Is this called direct by travis?  If not, it should be on the first line like all the other files (just convention)